app/api/ticket_fees.py

```diff
@@ -10,10 +12,35 @@ class TicketFeeList(ResourceList):
     """
     List and create TicketFees
     """
+    def before_post(self, args, kwargs, data):
+        """
+        before post method to check for existing currency-country combination
+        :param args:
+        :param kwargs:
+        :param data:
+        :return:
+        """
+        if 'country' in data and 'currency' in data:
+            if data['country'] and data['currency']:
+                try:
+                    TicketFees.query.filter_by(country=data['country'], currency=data['currency']).one()
+                except NoResultFound:
+                    pass
+                else:
+                    raise ConflictException(
+                        {'pointer': ''},
+                        "({}-{}) Combination already exists".format(data['currency'], data['country']))
+            else:
+                raise UnprocessableEntity({'source': ''}, "Country or Currency cannot be NULL")
+        else:
+            raise UnprocessableEntity({'source': ''}, "Country or Currency Attribute is missing")
```

Comment:
Use marshmallow for validation

Reply:
@iamareebjamal could you please elaborate?

Reply:
https://marshmallow.readthedocs.io/en/3.0/

Reply:
Understood, shall commit soon

Reply:
@uds5501 I think here @iamareebjamal simply means that use `allow_none=False` in the schema itself. It will raise error if country or currency is missing and you can use `nullable=False` in model to disallow use of NULL values.

Reply:
@shreyanshdwivedi Ooooh! I understand now. I will update it soon